compass/ocean/tests/global_ocean/global_ocean.cfg

```diff
@@ -21,7 +21,7 @@ sweep_count = 20
 [ssh_adjustment]
 
 # the number of iterations of ssh adjustment to perform
-iterations = 10
+iterations = 4
```

Comment:
Yay!

Reply:
Each iteration is slower, though. But hopefully this is a pretty big net speed-up.